### PR TITLE
1 Setting up proxy logic

### PR DESCRIPTION
**Pull request by Blynqs**

### CODE REVIEW

File: Middlewares/ProxyMiddleware.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    public ProxyMiddleware(RequestDelegate next)
+    {
+        _next = next;
+        _httpClient = new HttpClient();
+    }
+
+    public async Task InvokeAsync(HttpContext context)
+    {
+        string remoteIpAddress = context.Connection.RemoteIpAddress?.ToString();
```
Comment (LucasTesnier) [inline Middlewares/ProxyMiddleware.cs:14]: What happen if RemoteIpAddress is null ?

File: Middlewares/ProxyMiddleware.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        string userAgent = context.Request.Headers["User-Agent"];
```
Comment (LucasTesnier) [inline Middlewares/ProxyMiddleware.cs:15]: Is this gonna be change by @Cavonstavant appsettings logic provider ? (CF lucca)

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
Answer the questions
Btw, do we have a politic about in code documentation ?

File: Middlewares/ProxyMiddleware.cs (hunk 1 above)
Comment (Blynqs) [reply Middlewares/ProxyMiddleware.cs:14]: no exception will be thrown, unless you think it's important to handle it. Im open to any suggestion, but I forgot to mention this code is supposed to be temporary. This task is just the starting point for us to implement and perform new checks. 
Surely we will implement a better logging feature in the future this just shows how to access request information.

**Review by LucasTesnier — APPROVED** [review]

**Review by GuyomT — APPROVED** [review]
Good first implementation of proxy, you can merge

**Pull request merged**